test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -503,10 +503,10 @@ defmodule Membrane.RTC.HLSEndpointTest do
   defp check_correctness_of_output_files(output_dir, reference_dir) do
     output_files = output_dir |> File.ls!() |> Enum.sort()
 
-    output_manifests =
-      Enum.filter(output_files, fn file ->
-        String.match?(file, ~r/\.m3u8$/) and file != @main_manifest
-      end)
+    # output_manifests =
+    #   Enum.filter(output_files, fn file ->
+    #     String.match?(file, ~r/\.m3u8$/) and file != @main_manifest
+    #   end)
```

Comment:
If we don't use it shouldn't we remove it, or add some comments that this will be fixed in the future?